app/views/bookmarks/confirm_delete.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <!--main content-->
 <%= form_for(@bookmark, :html => {:method => :delete, :class => "simple destroy"}) do |f| %>
-  <p class="caution notice"><%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of "%{bookmarkable}"?', :bookmarkable => @bookmark.bookmarkable.title).html_safe %></p>
+  <p class="caution notice"><%= ts('Are you sure you want to <strong><em>delete</em></strong> your bookmark of "%{bookmarkable}"?', :bookmarkable => (@bookmark.bookmarkable.nil? ? "This has been deleted, sorry!": @bookmark.bookmarkable.title)).html_safe %></p>
```

Comment:
This defeats the purpose of having it be in ts() though and also inserting "This has been deleted" into this sentence would look very odd. Doesn't this option only come up ON the bookmark itself? Do we actually need to include the name of the bookmarkable object? I suggest we remove it.